src/pipx/animate.py

```diff
@@ -4,8 +4,8 @@
 from threading import Event, Thread
 from typing import Generator, List
 
+import pipx.emojis as emojis
```

Comment:
This change seems unrelated... I prefer to not mix changes, as it gets unclear why they were introduced. Also, is it necessary?